frost-coordinator/src/coordinator.rs

```diff
@@ -469,51 +469,131 @@ where
 
 #[cfg(test)]
 mod test {
+    use crate::{
+        coordinator::{Command, Coordinator},
+        DEVNET_COORDINATOR_ID,
+    };
+    use frost_signer::{
+        config::{Config, PublicKeys, SignerKeyIds},
+        logging,
+        net::{HttpNet, HttpNetListen},
+        signer::Signer,
+    };
     use hashbrown::HashMap;
-    use test_utils::Process;
+    use p256k1::{ecdsa, scalar::Scalar};
+    use rand_core::OsRng;
+    use relay_server::Server as RelayServer;
+    use std::{env, thread};
+
+    fn create_signer_key_ids(signer_id: u32, keys_per_signer: u32) -> Vec<u32> {
+        let mut signer_key_ids = Vec::new();
+        for i in 0..keys_per_signer {
+            signer_key_ids.push(keys_per_signer * signer_id + i + 1);
+        }
+        signer_key_ids
```

Comment:
Nit: imo it's cleaner to use iterators whenever possible to construct collections
```suggestion
        (0..keys_per_signer)
            .map(|i| keys_per_signer * signer_id + i + 1)
            .collect()
```

Reply:
Done.